### DISCUSSION TITLE
Documentation needs to be corrected

### DISCUSSION THREAD

**Discussion by MrOxMasTer** [statelyai/docs#444] [Q&A] [answered]
I only see errors in it when trying to write by example. The topic is really new for many developers, as well as for me. But it is simply unbearable with such documentation.  You can't do anything beyond the simplest automaton. As soon as you want to link automata, nothing will work, because the documentation does not tell you how to do it and in what situations to do it, and if it does, it is very confusing and it will not work

Mistakes just don't get resolved for months, considering there aren't many of them at all:
https://github.com/statelyai/docs/issues?q=is%3Aopen+is%3Aissue

https://github.com/statelyai/docs/issues/442
https://github.com/statelyai/docs/issues/440

It is not clear how to link the automata to each other at all. Because there are 2 ways: call or create.
Nobody will tell you which one you need and what “system of automata” means and how it differs from “call” or “create”, and when you try to write this very “system”, a question arises. And how to add another automaton, for example, because invoke `systemId` has only one and `src` too

The most frustrating thing is that it doesn't even display normally and you don't realize if you wrote it right or wrong at all:
![image](https://github.com/user-attachments/assets/fec6c4df-f662-48fe-808b-5e9395e2071d)

Very big questions about the organization of the documentation and specifically these articles where the examples don't work:
https://stately.ai/docs/spawn
https://stately.ai/docs/system

**Comment by davidkpiano** [comment, marked as answer]
Very sorry about that. I will review the documentation and ensure the examples are up-to-date.

**Reply by MrOxMasTer** [reply]
> Very sorry about that. I will review the documentation and ensure the examples are up-to-date.

I'm sorry if it's rude. I was very upset that I couldn't use your library.
